cli.ts

```diff
@@ -59,18 +59,21 @@ if (core.getBooleanInput("preprocess")) {
     console.log("Moved README.md to Home.md");
   }
 
-  const mdRe = /\.(?:md|markdown|mdown|mkdn|mkd|mdwn|mkdown|ron)$/;
+  const mdRe = /\.(?:md|markdown|mdown|mkdn|mkd|mdwn|mkdown|ron)(#[\w\-]*)?$/;
   const plugin = () => (tree: any) =>
     visit(tree, ["link", "linkReference"], (node: any) => {
-      if (!mdRe.test(node.url)) {
+      const matches = node.url?.match(mdRe)
+      if (matches === null || matches === undefined) {
```

Comment:
why not use `!`

Reply:
as in `!matches`? that could also work. :)